.gitignore

```diff
@@ -1,6 +1,7 @@
 # NPM instalation
 /node_modules/
 *npm-debug.log
+package-lock.json
```

Comment:
If you have a `package-lock.json` file, it means you used the wrong command :)

Reply:
`npm link` to link the "bin" section of the package.json, which yarn doesn't do :')